### PR TITLE
sestatus: add page

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

https://github.com/tldr-pages/tldr/issues/11896

### CODE REVIEW

**Comment by Managor** [comment]
`-v` and `-b` also exist but I can´t test them because I don't have the environment. I assume `-b` does the same as what `sudo semanage boolean --list | head` does https://www.redhat.com/sysadmin/change-selinux-settings-boolean

**Review by cyqsimon — APPROVED** [review]
`sestatus -b` seems useful as well, but it's up to you.

Edit: sorry, didn't see your latest comment.

Yeah `semanage boolean --list`'s output actually shows more details than `sestatus -b`:

```bash
# semanage boolean --list | head
SELinux boolean                State  Default Description

abrt_anon_write                (off  ,  off)  Allow abrt to anon write
abrt_handle_event              (off  ,  off)  Allow abrt to handle event
abrt_upload_watch_anon_write   (on   ,   on)  Allow abrt to upload watch anon write
antivirus_can_scan_system      (off  ,  off)  Allow antivirus to can scan system
antivirus_use_jit              (off  ,  off)  Allow antivirus to use jit
auditadm_exec_content          (on   ,   on)  Allow auditadm to exec content
authlogin_nsswitch_use_ldap    (off  ,  off)  Allow authlogin to nsswitch use ldap
authlogin_radius               (off  ,  off)  Allow authlogin to radius

# sestatus -b | head -n 20
SELinux status:                 enabled
SELinuxfs mount:                /sys/fs/selinux
SELinux root directory:         /etc/selinux
Loaded policy name:             targeted
Current mode:                   enforcing
Mode from config file:          enforcing
Policy MLS status:              enabled
Policy deny_unknown status:     allowed
Memory protection checking:     actual (secure)
Max kernel policy version:      33

Policy booleans:
abrt_anon_write                             off
abrt_handle_event                           off
abrt_upload_watch_anon_write                on
antivirus_can_scan_system                   off
antivirus_use_jit                           off
auditadm_exec_content                       on
authlogin_nsswitch_use_ldap                 off
authlogin_radius                            off
```

And `sestatus -v` FYI:

```bash
# sestatus -v
SELinux status:                 enabled
SELinuxfs mount:                /sys/fs/selinux
SELinux root directory:         /etc/selinux
Loaded policy name:             targeted
Current mode:                   enforcing
Mode from config file:          enforcing
Policy MLS status:              enabled
Policy deny_unknown status:     allowed
Memory protection checking:     actual (secure)
Max kernel policy version:      33

Process contexts:
Current context:                unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023
Init context:                   system_u:system_r:init_t:s0
/sbin/agetty                    system_u:system_r:getty_t:s0-s0:c0.c1023
/usr/sbin/sshd                  system_u:system_r:sshd_t:s0-s0:c0.c1023

File contexts:
Controlling terminal:           unconfined_u:object_r:user_devpts_t:s0
/etc/passwd                     system_u:object_r:passwd_file_t:s0
/etc/shadow                     system_u:object_r:shadow_t:s0
/bin/bash                       system_u:object_r:shell_exec_t:s0
/bin/login                      system_u:object_r:login_exec_t:s0
/bin/sh                         system_u:object_r:bin_t:s0 -> system_u:object_r:shell_exec_t:s0
/sbin/agetty                    system_u:object_r:getty_exec_t:s0
/sbin/init                      system_u:object_r:bin_t:s0 -> system_u:object_r:init_exec_t:s0
/usr/sbin/sshd                  system_u:object_r:sshd_exec_t:s0
```

**Comment by Managor** [comment]
What does `sestatus -bv` show?

**Comment by cyqsimon** [comment]
> What does `sestatus -bv` show?

All sections. Main section first, then booleans, then process contexts, then file contexts.

```bash
SELinux status:                 enabled
SELinuxfs mount:                /sys/fs/selinux
SELinux root directory:         /etc/selinux
Loaded policy name:             targeted
Current mode:                   enforcing
Mode from config file:          enforcing
Policy MLS status:              enabled
Policy deny_unknown status:     allowed
Memory protection checking:     actual (secure)
Max kernel policy version:      33

Policy booleans:
abrt_anon_write                             off
abrt_handle_event                           off
abrt_upload_watch_anon_write                on
antivirus_can_scan_system                   off
antivirus_use_jit                           off
auditadm_exec_content                       on
...

Process contexts:
Current context:                            unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023
Init context:                               system_u:system_r:init_t:s0
/sbin/agetty                                system_u:system_r:getty_t:s0-s0:c0.c1023
/usr/sbin/sshd                              system_u:system_r:sshd_t:s0-s0:c0.c1023

File contexts:
Controlling terminal:                       unconfined_u:object_r:user_devpts_t:s0
/etc/passwd                                 system_u:object_r:passwd_file_t:s0
/etc/shadow                                 system_u:object_r:shadow_t:s0
/bin/bash                                   system_u:object_r:shell_exec_t:s0
/bin/login                                  system_u:object_r:login_exec_t:s0
/bin/sh                                     system_u:object_r:bin_t:s0 -> system_u:object_r:shell_exec_t:s0
/sbin/agetty                                system_u:object_r:getty_exec_t:s0
/sbin/init                                  system_u:object_r:bin_t:s0 -> system_u:object_r:init_exec_t:s0
/usr/sbin/sshd                              system_u:object_r:sshd_exec_t:s0
```

For completeness's sake, `-vb` instead of `-bv` does not affect section ordering.

**Comment by Managor** [comment]
Alright, thank you.

**Comment by Managor** [comment]
Let me know if that wording in is off in any way.

File: pages/linux/sestatus.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# sestatus
+
+> Print the current SELinux status.
+> More information: <https://manned.org/sestatus>.
+
+- Print the current status:
+
+`sestatus`
+
+- Print the current boolean state:
```
Comment (cyqsimon) [inline pages/linux/sestatus.md:10]:
```suggestion
- Print the current states of all policy booleans:
```

Maybe this is clearer. Otherwise LGTM.

**Pull request merged**